app/cdap/components/AbstractWidget/FormInputs/TextBox/index.tsx

```diff
@@ -72,6 +83,9 @@ const TextBoxComponent: React.FC<ITextBoxProps> = ({
       onKeyPress={onKeyPress}
       placeholder={placeholder}
       readOnly={disabled}
+      multiline={multiline}
+      minRows={minRows}
```

Comment:
This and maxRows only apply when multiline is `true`. Won't affect current textboxes.